doc/instanceVisualization.md

```diff
@@ -425,11 +443,19 @@ Check the function **URL** and **secret** in the front-end configuration file.
 
 Check the error log in your Function App to get more information on the error.
 
-If the error message is _“Access is denied”_:
+If the error message is **_“Access is denied”_**:
 
 - check that your app registration has the permissions `Dataset.Reader`, `Scenario.Reader` and `Workspace.Reader`
 - check that these permissions have been granted by an administrator
 
+If the error message complains about a **scenario not found**:
```

Comment:
**_scenario not found_**: seems better with other if messages

Reply:
The problem is that unlike the two other error messages, this "scenario not found" is not an exact quote, the error message tends to change based on the API version used. That's why I preferred a more vague sentence with "complains about ..." :smile: